packages/loaders/src/elements/Dots.tsx

```diff
@@ -61,18 +61,18 @@ export const Dots = forwardRef<SVGSVGElement, IDotsProps>(
 
     return (
       <StyledSVG
+        $dataGardenId={COMPONENT_ID}
```

Comment:
```suggestion
        dataGardenId={COMPONENT_ID}
```

The `data-garden-id` needs to remain intact in the DOM

Reply:
`dataGardenId` would be filtered out by [@emotion/is-prop-valid](https://www.npmjs.com/package/@emotion/is-prop-valid) used under the hood in `styled-components@v5`. I've replaced it with `data-garden-id`.

![Screenshot 2024-11-06 at 8 03 16 AM](https://github.com/user-attachments/assets/25062b3b-c5a6-42df-b4e4-ff2986d6ab9f)

Reply:
Hmm, bit of a bummer about that filtering.